docs/sources/mimir/configure/configuration-parameters/index.md

```diff
@@ -3284,6 +3284,12 @@ The `limits` block configures default and per-tenant limits imposed by component
 # CLI flag: -ingester.native-histograms-ingestion-enabled
 [native_histograms_ingestion_enabled: <boolean> | default = false]
 
+# (experimental) Enable experimental out-of-order native histogram ingestion.
+# This will only take effect if ingester.out-of-order-time-window is > 0 and if
+# ingester.native-histograms-ingestion-enabled = true
```

Comment:
```suggestion
# `ingester.native-histograms-ingestion-enabled = true`.
```

Reply:
Updated in https://github.com/grafana/mimir/pull/7175/commits/0887714a2e152a807576471c76e0df34f1d97e46#diff-a81dd9ab4b082f2382b9752c72f5362dc02243116164bbad99cfb2ec799230bf